website_sale_product_attachment/views/product_template.xml

```diff
@@ -16,7 +17,8 @@
                         or unpublish it from your database.
                     </div>
                     <field name="website_attachment_ids" nolabel="1">
-                        <tree>
+                        <tree default_order="website_name,name,id">
```

Comment:
This will produce different orders in backend and frontend. Is that what you want?

![imagen](https://user-images.githubusercontent.com/973709/134884237-706bcaea-c7a7-4c83-abfd-1c171464d3c6.png)

To avoid that, `website_name` should be a computed writable field, and order on that only. Or a `sequence` field could be used instead.

Reply:
Although you are right, I prefer to not over-complicate the design for that only case where you don't fill one of the fields, as we can only sort by stored fields, and having 3 fields for such simple feature is too much (and think also if adding translations. This simply won't work).